app/config/config.yml

```diff
@@ -136,3 +136,10 @@ prestashop:
         api_client:
             ttl: 7200 # 2h
             # verify_ssl: ~ # Bundle CA by default, declaring "addons.api_client.verify_ssl" parameter overrides its usage
+
+tactician:
```

Comment:
if this middleware only needs to be enabled in dev mode, how about put this configuration in `config_dev.yml` file instead? This may remove the check about the application environment in the class.

Reply:
Yeah, that would make sense.